templates/download/iot/intel-iotg.html

```diff
@@ -53,14 +53,14 @@ <h3>Ubuntu Core 20</h3>
       <p>
         <a class="p-button--positive" href="https://cdimage.ubuntu.com/releases/focal/release/inteliotg/ubuntu-core-20-amd64+intel-iot.img.xz">Download</a>
       </p>
-      <p>Please see the <a href="https://assets.ubuntu.com/v1/377806cd-October_25-Release+Notes.pdf">release notes</a> and <a href="https://assets.ubuntu.com/v1/87e46aa7-Installation+Instructions+Core+Image+Intel+IoTG+Platforms.pdf">instructions</a> for more information.</p>
+      <p>Please see the <a href="https://assets.ubuntu.com/v1/ab850656-November_24-Release+Notes.pdf">release notes</a> and <a href="https://assets.ubuntu.com/v1/87e46aa7-Installation+Instructions+Core+Image+Intel+IoTG+Platforms.pdf">instructions</a> for more information.</p>
     </div>
     <div class="col-6 p-divider__block">
       <h3>Ubuntu Desktop 20.04</h3>
       <p>
-        <a class="p-button--positive" href="https://cdimage.ubuntu.com/releases/focal/release/inteliotg/ubuntu-20.04-preinstalled-desktop-amd64+intel-iot.img.xz">Download</a>
+        <a class="p-button--positive" href="https://cdimage.ubuntu.com/releases/focal/release/inteliotg/focal-desktop-amd64+intel-iot-20211110.5.iso">Download</a>
```

Comment:
We got the link earlier this evening ([copy doc](https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit?disco=AAAASXJyg8Q))
```suggestion
        <a class="p-button--positive" href="https://cdimage.ubuntu.com/releases/focal/release/inteliotg/ubuntu-20.04-desktop-amd64+intel-iot.iso">Download</a>
```